app/src/main/java/com/aileyzhang/musicapp/SongData.java

```diff
@@ -1,52 +1,71 @@
 package com.aileyzhang.musicapp;
 
+import android.content.Context;
+import android.database.Cursor;
 import android.graphics.Bitmap;
 import android.graphics.BitmapFactory;
 import android.media.MediaMetadataRetriever;
 import android.media.MediaPlayer;
+import android.net.Uri;
 import android.os.Environment;
+import android.provider.MediaStore;
 
+import java.io.ByteArrayInputStream;
 import java.io.File;
+import java.io.InputStream;
+import java.net.URI;
 import java.util.ArrayList;
 
+import static android.graphics.BitmapFactory.decodeFile;
+
 /**
  * Created by Ailey on 2017-12-15.
  */
 
 public class SongData {
-    private static final String MEDIA_PATH = Environment.getExternalStorageDirectory().getPath() +
-            File.separator + "Download" + File.separator;
-    private static final String mp3Pattern = ".mp3";
+    private Context context;
     private MediaPlayer mMediaPlayer = new MediaPlayer();
```

Comment:
Unused, please remove and readd when you actually use it (don't commit testing variables, etc). You will probably have another class to actually manage playback instead of doing it from here as you probably know 👍